foundry.toml

```diff
@@ -12,5 +12,8 @@ via_ir = true
 optimizer = true
 optimizer_runs = 100000000
 
+[fmt]
+ignore = ["src/vendor/**/*"]
+
```

Comment:
We want to ignore vendored contracts from being formatted to limit the amount of the diffs from the source.